feat(nextjs): Add Next.js docs (#3319)
getsentry/sentry-docs: src/platforms/javascript/guides/nextjs/manual-setup/run-dev.mdx

---

> title: Running in development

Reviewer: Suggest moving this page to Configuration
```suggestion
title: Default Mode
```

Reviewer 2: Hmm, I suspect Next.js' development mode is not the same as "default mode"?
The "Capitalized Title Suggestion" is good, that's our [style guideline](https://www.notion.so/sentry/Capitalization-eb9c6f9dd5614cb9b9f56140f06189c4#2af991efc6474c6f94134dcfbaa08034) for titles.

Author: No. I think "Development Mode" should do it.

sidebar_order: 4
description: "Learn how to enable the SDK to run in development mode."
---

> Next.js’ development mode doesn’t perform some checks the production mode does during the build process, and there may be errors that aren’t caught (including these that would prevent a successful build) and make the SDK not behave as expected.
> Thus, the Next.js SDK is only enabled by default in production environments.

Reviewer: ```suggestion
Sentry's Next.js SDK is enabled by default in production environments. This default mode is because Next.js’ development mode doesn’t perform some checks as it does in production mode during the build process. As a result, some errors may not be caught (including these that would prevent a successful build) and the SDK may not behave as expected.
```

## Enabling the SDK in development

> You may want to enable the SDK in other environments like development. However, note that enabling the SDK in development doesn’t change its behavior.

Reviewer: ```suggestion
You can enable the SDK in other environments, such as development; however, enabling the SDK in development doesn’t change its behavior.
```
